csrankings.csv

```diff
@@ -403,6 +404,7 @@ Alessandro Moschitti,Qatar Computing Research Institute,http://www.qcri.org/our-
 Alessandro Orso,Georgia Institute of Technology,http://www.cc.gatech.edu/home/orso,wCfYkMkAAAAJ
 Alessandro Redondi,Politecnico di Milano,http://home.deib.polimi.it/redondi,8ka5NmUAAAAJ
 Alessandro Salandrino,University of Kansas,http://www.eecs.ku.edu/people/faculty,ZX0I5pcAAAAJ
+Alessandro Sperduti,University of Padova,https://www.math.unipd.it/~sperduti,JsExaWMAAAAJ

```

Comment:
Department of Math is not Computer Science.

Reply:
Unless this person has a cross-appointment in CS and can thus solely advise a PhD student _for a CS PhD_, they are not eligible for inclusion.

Reply:
This is a Math & CS Department (in Italy this is not the only example). All the faculties from Univ. of Padova that are included here can advise PhD students and they are teaching in the bachelor, master and PhD program in computer science.